css/obBase.css

```diff
@@ -3620,8 +3621,8 @@ input[type="checkbox"].fieldItem:checked + label .togLabelOff {
 
 .spinner-with-logo i {
```

Comment:
It would be better to apply the padding in a rule with a selector based on this class, like .spinner-with-logo i:before

I'll move the property in a patch update.